app/dao/service_broadcast_settings_dao.py

```diff
@@ -0,0 +1,15 @@
+from app import db
+from app.models import ServiceBroadcastSettings
+
+
+def insert_or_update_service_broadcast_settings(service, channel, provider_restriction=None):
```

Comment:
Does this method need to be in a different file? 
Can service_broadcast_settings_dao and broadcast_service_dao be merged into one file?